src/app/api/geolocation/route.ts

```diff
@@ -0,0 +1,69 @@
+import { geolocation } from "@vercel/functions";
+import { NextResponse, type NextRequest } from "next/server";
+
+// All EU/EEA countries with GDPR codes
+// source: https://www.gdpradvisor.co.uk/gdpr-countries
+const GDPR_COUNTRIES = new Set([
+  "AT", // Austria
+  "BE", // Belgium
+  "BG", // Bulgaria
+  "HR", // Croatia
+  "CY", // Cyprus
+  "CZ", // Czech Republic
+  "DK", // Denmark
+  "EE", // Estonia
+  "FI", // Finland
+  "FR", // France
+  "DE", // Germany
+  "GR", // Greece
+  "HU", // Hungary
+  "IS", // Iceland (EEA)
+  "IE", // Ireland
+  "IT", // Italy
+  "LV", // Latvia
+  "LI", // Liechtenstein (EEA)
+  "LT", // Lithuania
+  "LU", // Luxembourg
+  "MT", // Malta
+  "NO", // Norway (EEA)
+  "NL", // Netherlands
+  "PL", // Poland
+  "PT", // Portugal
+  "RO", // Romania
+  "SK", // Slovakia
+  "SI", // Slovenia
+  "ES", // Spain
+  "SE", // Sweden
+]);
+
+const corsHeaders = {
+  "Access-Control-Allow-Origin": "*",
+  "Access-Control-Allow-Methods": "GET",
+  "Access-Control-Allow-Headers": "Content-Type",
+  "Access-Control-Max-Age": "86400",
+};
```

Comment:
why?

Reply:
Since i can't have api route on docs page, I send request to upstash.com's endpoint. And there was a CORS issue between them. That's why I needed to do this CORS header thing